crates/shared/src/price_estimation.rs

```diff
@@ -105,8 +105,8 @@ impl Estimate {
     }
 }
 
-#[mockall::automock]
 #[async_trait::async_trait]
+#[mockall::automock]
```

Comment:
Funny how significant this small change actually is!

Reply:
To be honest I'm a little surprised that the old version even worked considering that async traits aren't stable yet.
BTW they specifically mention `async_trait` in their [docs](https://docs.rs/mockall/latest/mockall/#async-traits).